src/pages/AlumniPage.jsx

```diff
@@ -0,0 +1,84 @@
+            <img src={PlaceholderPic} alt="Placeholder" className="w-full max-w-[1920px] h-auto mt-8"/>
+
+            <div className="max-w-[1088px] w-full flex flex-col justify-center items-center gap-10 m-10">
+                <h1 className="text-black text-center font-inter text-4xl font-medium" style={{ letterSpacing: '8px' }}>
+                    HOW TO GET INVOLVED
+                </h1>
+                <p className="text-black text-center font-inter text-base leading-7 px-32">
+                    We strongly encourage all FUSION alumni to stay connected! There are many ways to stay involved and still make an impact on the FUSION space.
+                </p>
+            </div>
+
+            <div className="flex items-start max-w-[1270px] w-full h-[413px] mt-4">
+                <img src={PlaceholderPic} alt="Scholarship Placeholder" className="w-[626px] h-[413px] opacity-1"/>
+                <div className="ml-8 flex flex-col justify-center">
+                    <h2 className="text-black font-inter text-4xl font-medium pr-40" style={{ letterSpacing: '8px' }}>
+                        ALUMNI SCHOLARSHIP
+                    </h2>
+                    <p className="text-black font-inter text-base leading-7 pr-32 mt-4">
+                        Help us fund a yearly scholarship for a FUSION general member that has actively contributed to the community!
+                    </p>
+                </div>
+            </div>
+
+            <div className="flex items-start max-w-[1270px] w-full h-[413px] mt-10">
+                <div className="mr-8 flex flex-col justify-center">
+                    <h2 className="text-black font-inter text-4xl font-medium" style={{ letterSpacing: '8px' }}>
+                        PROFESSIONAL NETWORK
+                    </h2>
+                    <p className="text-black font-inter text-base leading-7">
+                        Provide your contact information and a little bit about your career path to allow members interested in your career to reach out!
+                    </p>
+                </div>
+                <img src={PlaceholderPic} alt="Network Placeholder" className="w-[626px] h-[413px] opacity-1"/>
+            </div>
+
+            <div className="flex items-start max-w-[1270px] w-full h-[413px] mt-10">
+                <img src={PlaceholderPic} alt="Spotlight Placeholder" className="w-[626px] h-[413px] opacity-1"/>
+                <div className="ml-8 flex flex-col justify-center">
+                    <h2 className="text-black font-inter text-4xl font-medium pr-40" style={{ letterSpacing: '8px' }}>
+                        ALUMNI SPOTLIGHT
+                    </h2>
+                    <p className="text-black font-inter text-base leading-7">
+                        The Networking Directors will spotlight an Alumni each quarter, so let us know if you’d like to be highlighted or would like to recommend someone.
+                    </p>
+                </div>
+            </div>
```

Comment:
let's make the spacing a little bit more spreadout in terms of the components within the How To Get Invovled